dashboard/public/frequency/frequency.html

```diff
@@ -0,0 +1,61 @@
+<link type="text/css" rel="stylesheet" href="/frequency/frequency.css">
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
```

Comment:
I do ultimately want to remove D3 as a dependency; I mostly included it because it made early refactoring a lot easier.